### PR TITLE
added ordering to python versions endpoint

### PR DESCRIPTION
**Pull request by Gkrumbach07**
## Related Issues and Dependencies
https://github.com/thoth-station/user-api/issues/1670

## This introduces a breaking change

- No

## This Pull Request implements

Added endpoint support for ordering request by ASC or DESC.

## Description

A new optional param is added to the `/python/package/versions` endpoint named `order_by`. It takes `ASC` or `DESC` to order the output in ascending order or descending order.

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -972,6 +972,7 @@ paths:
       parameters:
       - $ref: "#/components/parameters/page"
       - $ref: "#/components/parameters/per_page"
+      - $ref: "#/components/parameters/order_by"
```
Comment (harshad16) [inline openapi/openapi.yaml:975]: Please add parameter description in the component defination
https://github.com/thoth-station/user-api/blob/b80e0d55faa9a88c67119ee235a81dd28e9954e9/openapi/openapi.yaml#L1104

**Review by harshad16 — CHANGES_REQUESTED** [review]
one minor change needed

File: openapi/openapi.yaml (hunk 1 above)
Comment (Gkrumbach07) [reply openapi/openapi.yaml:975]: added description

File: thoth/user_api/api_v1.py [hunk 2]
```diff
@@ -731,6 +732,11 @@ def list_python_package_versions(
     page_count = ceil(entries_count / per_page)
     start_offset = _compute_offset(page=page, per_page=per_page, page_count=page_count)
 
+    if order_by and order_by in ["ASC", "DESC"]:
+        page_count = 1
+        start_offset = 0
+        per_page = entries_count
```
Comment (fridex) [inline thoth/user_api/api_v1.py:738]: You can pass directly `count=None` which will retrieve all the results in the database.

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -1145,6 +1146,17 @@ components:
...
       schema:
         type: string
+    order_by:
+      name: order_by
+      in: query
+      required: false
+      description: Sort the response list
+      schema:
+        type: string
+        nullable: true
```
Comment (fridex) [inline openapi/openapi.yaml:1156]:
```suggestion
```

**Review by fridex — COMMENTED** [review]
Looks good to me. What about also dropping `page` and `per_page`? It looks like they do not have use now.

**Review by fridex — APPROVED** [review]
Perfect, looks good to me. Thanks! 💯

**Pull request merged**